buffalo/cmd/buffalo.go

```diff
@@ -0,0 +1,77 @@
+// Copyright © 2016 NAME HERE <EMAIL ADDRESS>
```

Comment:
Yes, I copy pasted from another file of the project (I suspect dev.go https://github.com/gobuffalo/buffalo/blob/272bb3a0ab85628caeac9f3ef9e6f0a71f71ef2f/buffalo/cmd/dev.go#L1 ).
My question is (I suck at licensing): should I put my name here, since I wrote part of the code? Or Mark Bates since he is the maintainer?
Another question, the LICENSE at the root indicates it is MIT, but it is a header of apache. So which one should I use?